pkg/cnb/builder_builder.go

```diff
@@ -54,10 +54,11 @@ type builderBlder struct {
 	additionalLabels  map[string]string
 }
 
-func newBuilderBldr(kpackVersion string) *builderBlder {
+func newBuilderBldr(kpackVersion string, runImage string) *builderBlder {
```

Comment:
Should this remain in AddStack or a `RunImage` method? I think we were trying to keep `BuilderBldr` expressive.